english/watched/buy-cookies/bakery.md

```diff
@@ -0,0 +1 @@
+But the bakery is far away, and you can't find a taxi. So you hop on your bike.
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.